### ISSUE TITLE
Doc updates to support Google Analytics 4

### ISSUE THREAD

**Issue by bmtcril** [overhangio/tutor#837]
**Is your feature request related to a problem? Please describe.**
The docs currently have some references to enabling the legacy version of GA as a plugin. I'd like to update them for the new Google Analytics 4 configuration, or write new docs about how to enable the feature if you'd like to move away from using that as an example. The existing GA4 configuration docs are here, they're not much different: https://docs.openedx.org/en/latest/site_ops/how-tos/google-analytics.html

**Describe the solution you'd like**
Just looking for guidance on if/how maintainers would like to see these docs updated.

**Comment by regisb** [comment]
Yes, we will need to update this section: https://docs.tutor.overhang.io/plugins/examples.html#enable-google-analytics

Based on the upstream docs, I would expect a plugin to look as follows:

```
from tutor import hooks

  hooks.Filters.ENV_PATCHES.add_items([
      (
          "openedx-common-settings",
          "GOOGLE_ANALYTICS_4_ID = 'MY-MEASUREMENT-ID'"
      ),
      (
          "mfe-lms-common-settings",
          "MFE_CONFIG['GOOGLE_ANALYTICS_4_ID'] = 'MY-MEASUREMENT-ID'"
      ),
  ])
```

The second patch will depend on this PR to be merged: https://github.com/overhangio/tutor-mfe/pull/124

**Comment by regisb** [comment]
Closed by https://github.com/overhangio/tutor/pull/842
